api/test/wfl/unit/mime_type_test.clj

```diff
@@ -0,0 +1,47 @@
+(ns wfl.unit.mime-type-test
+  (:require [clojure.test :refer [deftest is testing]]
+            [wfl.mime-type :as mime-type]
+            [wfl.tools.workflows :as workflows]))
+
+(deftest test-mime-types
+  (let [cases [[".pdf" "application/pdf"]
+               [".txt" "text/plain"]
+               [".bam" "application/octet-stream"]
+               [".cram" "application/octet-stream"]
+               [".vcf" "text/plain"]
+               [".gz" "application/gzip"]
+               [".fasta" "application/octet-stream"]
+               [".html" "text/html"]]]
+    (doseq [[filename expected] cases]
+      (testing filename
+        (is (= expected (mime-type/ext-mime-type filename)))))))
+
+(defn ^:private get-files
```

Comment:
This function has the same "shape" as `ingest` in `wfl.integration.data-repo-test`. It would be nice to extract this logic. Not clear how to do this yet.